### DISCUSSION TITLE
Revamp the idea of 'Cheats' in vanilla.

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#4231] [Ideas]
I'm not entirely sure on the best solution here but I'm suggesting revamping how the `Cheats` Commands.dat argument works in vanilla.

From a player's perspective when I see 'cheats enabled' on a server it makes me think it's a creative server where any players have access to cheats, which usually isn't the case.

Most servers use some kind of plugin framework or module, be it rocket, openmod, etc that overrides permissions. It's weird/misleading to have only a couple of the vanilla commands behind the 'cheats' flag when other commands like /i, /v, etc from rocket are still allowed. I think it would make more sense to let server owners specify themselves (like monetization policy), or maybe even completely remove it.

Would like to hear other people's inputs as well but I think a lot of new server owners get confused about the `Cheats` thing and especially with the new menu changes it's somewhat inconsistent when looking at servers running plugin frameworks or modules.

**Comment by SDGNelson** [comment]
You're right that for the meantime it is most useful if looking for vanilla servers with some precaution against admin abuse. Considering that it *can* be useful in that case I'd be hesitant to remove it. It's unfortunate that we can't enforce it for modded servers. :/

**Reply by sarshane** [reply]
I think the simplest solution is to remove the cheats option, because at the moment it works illogically, only working for **5 commands** and disabling them **only for administrators**, since in most cases players do not have access to them. Also, I don’t really understand the situation with the **new filters**, where this **function is disabled in standard filters**, which forces the servers to use plugins that **replace the commands** disabled by the cheats option.
